build/package/host/etc/cvdr.toml

```diff
@@ -25,6 +25,12 @@
 # files forever.
 # KeepLogFilesDays = 30
 
+# Source for the Build API credentials to be used on cvd create operations.
+# Possible values are:
+#  - "injected": Use credentials stored in the server.
+#  - "none": Don't use Build API credentials.
```

Comment:
should we use `"empty": Use empty string credentials`? From what I gather in the code it seems the "none" concept ends up returning an empty string as the credential value. 

nit: let's also move either "none" or "empty" to the first option as it's the trivial one.

Reply:
The fact that the code uses an empty string is an implementation detail. The real outcome of this setting is that no credentials are used to communicate with the build api.
Changed the order as requested.